styleguide/template.md

```diff
@@ -148,7 +148,10 @@ Examples:
 
 For more information about using this notation, see [Using cross reference](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
 
-> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to view the source for the API page you want to link to and find the ms.assetid value. Individual overload values are not shown in the source. We're working on having a better system in the future.
+There are two ways to find the UID:
+
+- View the source for the API page you want to link to and find the ms.assetid value. Note that individual overload values are not shown in the source.
```

Comment:
Individual overloads aren't shown by the xref.docs.microsoft.com tool either. Maybe the sentence about individual overloads should be moved to the previous paragraph or to a new paragraph after the list.

Reply:
You're right, @BillWagner. Ignore my comment, @mairaw